uk_bin_collection/uk_bin_collection/councils/SouthHollandDistrictCouncil.py

```
@@ -22,115 +12,38 @@ class CouncilClass(AbstractGetBinDataClass):
-            soup = BeautifulSoup(driver.page_source, "html.parser")
-
-            # Find the results container
-            waste_blocks = soup.select("div.mycollections div.waste")
 
-            for block in waste_blocks:
-                try:
-                    # Get the bin type
-                    heading = block.find("h3").get_text(strip=True)
-                    bin_type = heading.split(":")[0]  # e.g., "Refuse"
+        user_uprn = kwargs.get("uprn")
+        check_uprn(user_uprn)
```

Ruff (0.14.7)

`check_uprn` may be undefined, or defined from star imports

(F405)